content/integrations/kubernetes_state.md

```diff
@@ -49,3 +31,29 @@ The Kubernetes-state check does not include any event at this time.
 
 ### Service Checks
 The Kubernetes-state check does not include any service check at this time.
+
+## Troubleshooting
+
+If you have any questions about Datadog or a use case our [Docs](https://docs.datadoghq.com/) didn’t mention, we’d love to help! Here’s how you can reach out to us:
```

Comment:
Aren't we dropping all the boilerplate support links?

Reply:
yup, this article is going to be removed